src/data/security-quiz.ts

```diff
@@ -286,6 +286,16 @@ const securityQuiz = [
       "RSA, named after its creators (Rivest, Shamir, and Adleman), is an encryption technique that takes advantage of the fact that factoring large prime numbers is hard.",
     Link: "https://www.freecodecamp.org/news/understanding-encryption-algorithms/",
   },
+  {
+    Question: "Which of these is NOT a OWASP top 10 vulnerability?",
```

Comment:
It is important to include a year because is regularly updated. 

```suggestion
    Question: "Which of these was NOT a 2021 OWASP top 10 security vulnerability?",
```